### DISCUSSION TITLE
time out when enable `terrainandmovements`

### DISCUSSION THREAD

**Discussion by truong0235** [ORelio/Minecraft-Console-Client#2078] [Q&A]
everything works fine, but when i set `terrainandmovements=true` to use oreminer script, i get kicked with `internal exception: in.netty.handler.timerout.readtimeouteception` reason after a while

so how can i fix it?

(sorry bc of bad english)

**Comment by milutinke** [comment]
Which version you are playing on?
Anything above 1.17 is not supported yet.

**Reply by truong0235** [reply]
minecraft 1.16.5 with forge disabled

**Reply by milutinke** [reply]
Which MCC build are you using?
I've tested build 2013 on 1.16.5 spigot it worked perfectly.
What might be causing it is that the bot is trying to walk into not loaded chunks.
Maybe wait for our 1.18.2 implementation, a lot of work has been done on terrain movement.

**Reply by truong0235** [reply]
i use lastest release
idk if it cause on server side or not
im just standing and run an afk mine script
ping is still acceptable (200ms, stable until being disconnected)

any solution for this?

**Reply by milutinke** [reply]
> i use lastest release idk if it cause on server side or not im just standing and run an afk mine script ping is still acceptable (200ms, stable until being disconnected)
> 
> any solution for this?

Most likely because of the ping or the server.
You could try out my fork form #1943, the terrain movement has been improved.
